packages/palette/src/elements/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,34 @@
+import styled, { css } from "styled-components"
+import { Box, BoxProps } from "../Box"
+
+/** Visually hide element, but present in DOM */
+export const visuallyHiddenMixin = css`
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+`
+
+/** Reset to `inherit` all properties that cause visually-hidden */
+export const visuallyUnhiddenMixin = css`
+  clip: inherit;
+  clip-path: inherit;
+  height: inherit;
+  overflow: inherit;
+  position: inherit;
+  white-space: inherit;
+  width: inherit;
+`
+
+export interface VisuallyHiddenProps extends BoxProps {
+  show?: boolean
+}
+
+/** Visually hide element, but present in DOM */
+export const VisuallyHidden = styled(Box)<VisuallyHiddenProps>`
+  ${visuallyHiddenMixin}
+  ${({ show }) => show && visuallyUnhiddenMixin}
+`
```

Comment:
Just some helpers here for showing and hiding. Useful for the aria-live stuff, for instance.

Reply:
**question**: Is this particular cocktail of css attributes based on your past experience, or is it recommended as a best practice somewhere for this kind of show/hide-y stuff?  (I.e. using a clip path wouldn't have occurred to me 🙈 )

Reply:
This is from [The A11Y Project](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/) but you're right there should be a comment explaining that.

Reply:
Yeahh, adding the link in code would be good. Also I wonder if this module's name could be tweaked in a way to communicate it's utility. `HideAccessibly` doesn't roll off the tongue quite as well, but _something_ in that vein that helps describe the context. 

Last thing... that article is from 2013. Are we sure this is still the best way to achieve this?

Reply:
👍 

2020 :p https://css-tricks.com/inclusively-hidden/

`InclusivelyHidden` is a much more specific/better name too

Reply:
Another good article from WebAIM: https://webaim.org/techniques/css/invisiblecontent/